finances/src/main/java/life/qbic/projectmanagement/domain/finances/offer/OfferId.java

```diff
@@ -1,9 +1,10 @@
-package life.qbic.projectmanagement.finances.offer;
+package life.qbic.projectmanagement.domain.finances.offer;
 
 import java.util.Objects;
+import javax.persistence.AttributeConverter;
 
 /**
- * <b>Offer Id</b>
+ * <b>OfferPreview Id</b>
```

Comment:
Why OfferPreview? Do you want to differentiate between the preview offer ID and the offer ID? Why?

Reply:
its a search and replace side-effect. Again ...

Reply:
i adjusted it